modules/price/src/Price.php

```diff
@@ -63,4 +64,214 @@ public function __toString() {
     return $this->amount . ' ' . $this->currencyCode;
   }
 
+  /**
+   * Convert a \Drupal\commerce_price\Price object to a different currency.
+   *
+   * @param string $currency
+   *   A currency code.
+   * @param string $rate
+   *   A currency rate corresponding to the currency code.
+   *
+   * @return Price
```

Comment:
Any object name given as a return or param hint must be fully qualified (have the full namespace).